database/src/types.rs

```diff
@@ -266,6 +175,38 @@ pub struct BlockInfoV3 {
     pub long_term_weight: u64,
 }
 
+//---------------------------------------------------------------------------------------------------- OutputFlags
+bitflags::bitflags! {
+    /// TODO
+    ///
+    /// ```rust
+    /// # use std::borrow::*;
+    /// # use cuprate_database::{*, types::*};
+    /// // Assert Storable is correct.
+    /// let a = OutputFlags::NON_ZERO_UNLOCK_TIME;
+    /// let b = Storable::as_bytes(&a);
+    /// let c: OutputFlags = Storable::from_bytes(b);
+    /// assert_eq!(a, c);
+    /// ```
+    ///
+    /// # Size & Alignment
+    /// ```rust
+    /// # use cuprate_database::types::*;
+    /// # use std::mem::*;
+    /// assert_eq!(size_of::<OutputFlags>(), 4);
+    /// assert_eq!(align_of::<OutputFlags>(), 4);
+    /// ```
+    #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+    #[derive(Copy, Clone, Debug, PartialEq, PartialOrd, Eq, Ord, Hash, Pod, Zeroable)]
+    #[repr(transparent)]
+    pub struct OutputFlags: u32 {
+        /// No flags set.
+        const NONE = 0b0000_0000;
+        /// This output has a non-zero unlock time.
+        const NON_ZERO_UNLOCK_TIME = 0b0000_0001;
+    }
+}
```

Comment:
## Types

- Added `OutputFlags` with `bitflags!()` to replace our `u32` in `struct Output`
- Added `block_blob` in `struct VerifiedBlockInformation`

Reply:
`NONE` doesn't need to exist, we can just use `empty()`. The docs also say to avoid zero bit flags: https://docs.rs/bitflags/latest/bitflags/#zero-bit-flags